src/ucp/tag/eager_rcv.c

```diff
@@ -168,13 +168,20 @@ ucp_eager_common_middle_handler(ucp_worker_t *worker, void *data, size_t length,
 {
     ucp_eager_middle_hdr_t *hdr = data;
     ucp_recv_desc_t *rdesc      = NULL;
+    ucp_ep_h ep UCS_V_UNUSED;
     ucp_tag_frag_match_t *matchq;
     ucp_request_t *req;
     ucs_status_t status;
     size_t recv_len;
     khiter_t iter;
     int ret;
 
+    /* check UCS_PTR_MAP_KEY_INVALID to pass CI */
```

Comment:
what fails if not have this?

Reply:
gtests which create onesided endpoint and do tag send